cypress/pageobject/Facility/FacilityLocation.ts

```diff
@@ -1,4 +1,23 @@
 class FacilityLocation {
+  loadLocationManagementPage() {
+    cy.awaitUrl("/");
+    cy.intercept("GET", "**/api/v1/facility/**").as("getFacilities");
+    cy.get("[id='facility-details']").first().click();
```

Comment:
this first().click() can lead to flaky test, so select based on facility name itself, don't select random facility